python/pyspark/sql/types.py

```diff
@@ -815,6 +815,54 @@ def jsonValue(self) -> Dict[str, Any]:
 
     @classmethod
     def fromJson(cls, json: Dict[str, Any]) -> "StructType":
+        """
+        Constructs StructType from a scheme defined in json format
+
+        Parameters
+        ----------
+        json : Dict of str/any or a dict like object e.g. json object
```

Comment:
Dict -> dict, "dict like" -> dict-like. Maybe say a tiny bit more about what dict of str/any is

Reply:
done